source/publishing_data/02_harvesting_a_catalog/harvesters/ftp_with_meta_csv.rst

```diff
@@ -79,11 +79,52 @@ Accepted metadata columns
+    +===========+====================================================================================================================================================================================================================+
+    | ``true``  | Sets the **Geographic coverage** metadata for the dataset to **Automatic**. The geographic coverage is automatically computed based on the dataset content or on the domain's dataset default geographic coverage. |
+    +-----------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+    | ``false`` | Sets the **Geographic coverage** metadata for the dataset to the value for ``geographic_reference``.                                                                                                               |
+    +-----------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+The geographic_reference metadata
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The ``geographic_reference`` column defines the location used for the dataset geographic coverage, which means the **Geographic coverage** metadata for the dataset is set to **Specific**.
+This ``geographic_reference`` column contains an array of georeference unique identifiers representing locations.
+
+Georeference unique identifiers use the following syntaxes based on the reference:
+
++----------------+----------------------------------------------------------------+---------------------------------------------------------------------------------+
+| Reference      | Description                                                    | Syntax                                                                          |
++================+================================================================+=================================================================================+
+| world          | The dataset contains content about different countries         | ``world``                                                                       |
++----------------+----------------------------------------------------------------+---------------------------------------------------------------------------------+
+| country        | The dataset contains country-level content                     | ``world_{{country code}}``                                                      |
++----------------+----------------------------------------------------------------+---------------------------------------------------------------------------------+
+| lower division | The dataset contains content about a specific country division | ``world_{{country code}}_{{administrative-level}}_{{administrative division}}`` |
```

Comment:
In fact it should be `{{country code}}_{{administrative-level}}_{{administrative division}}` (without `world_` prefix) sorry if I was unclear on this!